### ISSUE TITLE
Screen2Auto scrooling

### ISSUE THREAD

**Issue by akulp** [shmykelsa/AAAD#271]
Samsung S20, no root, A13, AA version 10.8.634424. 
I used KingInstaller.

1. I can't scroll or manage the screen in any way (should there be a cursor?) when I open any application (via S2A launcher). But tap works. Also touch button works.

Within the S2A option, I also tried displaying the entire phone screen (screen mirror), but nothing works the same when I want to scroll. Tap also works.

But the screen is displayed correctly, if I operate via the phone, everything is visible on the car screen.

Is there something I don't know? Is something connected with "Alternative touch" (other setting menu).

(I also installed Fermata and in webrowser/YT all works fine, I can scroll and tap)

2. I don't know how to close S2A (from AA) in a way that my phone screen go back to portrait mode. If I change app from S2A to another inside AA, phone stays in landscape. Probably there is somehow possible to completely close S2A from AA? If I click "pause in S2A menu, phone jumps in portrait mode for a second and then immediately back to landscape...

Thank you.

**Comment by shmykelsa** [comment]
Unrelated to AAAD. See [FAQs](https://github.com/shmykelsa/AAAD#ive-downloaded-the-app-xxx-from-aaad-but-its-not-working-well-what-can-i-do) for more info.

**Comment by akulp** [comment]
> Unrelated to AAAD. See [FAQs](https://github.com/shmykelsa/AAAD#ive-downloaded-the-app-xxx-from-aaad-but-its-not-working-well-what-can-i-do) for more info.

Agree. But I thought someone had the same problem and can assist me.

I didn't found any GitHub discussion about S2A.
